### ISSUE TITLE
Crashes on New Immersive update!!!!!!

### ISSUE THREAD

**Issue by BedrockLegends** [Flaxbeard/ImmersivePetroleum#196]
after updating to new Immersive update, this mod crashes and wont load

**Comment by kzdaulet** [comment]
Yes i have that problem too, after update immersive ingeenering, petroleum can't start https://pastebin.com/1V2izknU

**Comment by b52src** [comment]
same https://pastebin.com/YbCaDz9p

**Comment by Radlongstaff** [comment]
You have to downgrade your IE version, not currently working with the newest update.

**Comment by drakray** [comment]
Same with https://pastebin.com/hzbXY0hP

<details><summary>Snippet for easy searching</summary>

```
Description: Mod loading error has occurred

java.lang.Exception: Mod Loading has failed
	at net.minecraftforge.fml.CrashReportExtender.dumpModLoadingCrashReport(CrashReportExtender.java:85) ~[forge:?] {re:classloading}
	at net.minecraftforge.fml.client.ClientModLoader.completeModLoading(ClientModLoader.java:188) ~[forge:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.lambda$null$1(Minecraft.java:508) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:charm.mixins.json:accessor.MinecraftAccessor,pl:mixin:APP:immersiveengineering.mixins.json:accessors.client.MinecraftAccess,pl:mixin:APP:assets/botania/botania.mixins.json:AccessorMinecraft,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.util.Util.func_215077_a(SourceFile:404) ~[?:?] {re:classloading}
	at net.minecraft.client.Minecraft.lambda$new$2(Minecraft.java:504) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:charm.mixins.json:accessor.MinecraftAccessor,pl:mixin:APP:immersiveengineering.mixins.json:accessors.client.MinecraftAccess,pl:mixin:APP:assets/botania/botania.mixins.json:AccessorMinecraft,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.gui.ResourceLoadProgressGui.func_230430_a_(ResourceLoadProgressGui.java:113) ~[?:?] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.GameRenderer.func_195458_a(GameRenderer.java:480) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.func_195542_b(Minecraft.java:976) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:charm.mixins.json:accessor.MinecraftAccessor,pl:mixin:APP:immersiveengineering.mixins.json:accessors.client.MinecraftAccess,pl:mixin:APP:assets/botania/botania.mixins.json:AccessorMinecraft,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:607) ~[?:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:charm.mixins.json:accessor.MinecraftAccessor,pl:mixin:APP:immersiveengineering.mixins.json:accessors.client.MinecraftAccess,pl:mixin:APP:assets/botania/botania.mixins.json:AccessorMinecraft,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:184) ~[1.16.4-forge-35.1.13.jar:?] {re:classloading,pl:runtimedistcleaner:A}
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_271] {}
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_271] {}
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_271] {}
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_271] {}
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider.lambda$launchService$0(FMLClientLaunchProvider.java:51) ~[forge-1.16.4-35.1.13.jar:35.1] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-8.0.6.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54) [modlauncher-8.0.6.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72) [modlauncher-8.0.6.jar:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:82) [modlauncher-8.0.6.jar:?] {re:classloading}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:66) [modlauncher-8.0.6.jar:?] {re:classloading}


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at flaxbeard.immersivepetroleum.common.multiblocks.DistillationTowerMultiblock.<init>(DistillationTowerMultiblock.java:23) ~[immersivepetroleum:3.1.0] {re:classloading}
-- MOD immersivepetroleum --
Details:
	Mod File: ImmersivePetroleum-1.16.3-3.1.0.jar
	Failure message: Immersive Petroleum (immersivepetroleum) encountered an error during the common_setup event phase
		java.lang.NoSuchMethodError: blusunrize.immersiveengineering.common.blocks.multiblocks.IETemplateMultiblock.<init>(Lnet/minecraft/util/ResourceLocation;Lnet/minecraft/util/math/BlockPos;Lnet/minecraft/util/math/BlockPos;Ljava/util/function/Supplier;)V
	Mod Version: 3.1.0
	Mod Issue URL: https://github.com/Flaxbeard/ImmersivePetroleum/issues
	Exception message: java.lang.NoSuchMethodError: blusunrize.immersiveengineering.common.blocks.multiblocks.IETemplateMultiblock.<init>(Lnet/minecraft/util/ResourceLocation;Lnet/minecraft/util/math/BlockPos;Lnet/minecraft/util/math/BlockPos;Ljava/util/function/Supplier;)V
Stacktrace:
	at flaxbeard.immersivepetroleum.common.multiblocks.DistillationTowerMultiblock.<init>(DistillationTowerMultiblock.java:23) ~[immersivepetroleum:3.1.0] {re:classloading}
	at flaxbeard.immersivepetroleum.common.multiblocks.DistillationTowerMultiblock.<clinit>(DistillationTowerMultiblock.java:20) ~[immersivepetroleum:3.1.0] {re:classloading}
	at flaxbeard.immersivepetroleum.common.IPContent.init(IPContent.java:170) ~[immersivepetroleum:3.1.0] {re:classloading}
	at flaxbeard.immersivepetroleum.ImmersivePetroleum.setup(ImmersivePetroleum.java:88) ~[immersivepetroleum:3.1.0] {re:classloading}
	at net.minecraftforge.eventbus.EventBus.doCastFilter(EventBus.java:247) ~[eventbus-3.0.5-service.jar:?] {}
	at net.minecraftforge.eventbus.EventBus.lambda$addListener$11(EventBus.java:239) ~[eventbus-3.0.5-service.jar:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:297) ~[eventbus-3.0.5-service.jar:?] {}
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.acceptEvent(FMLModContainer.java:120) ~[forge:35.1] {re:classloading}
	at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$4(ModContainer.java:121) ~[forge:?] {re:classloading}
	at java.util.concurrent.CompletableFuture$AsyncRun.run(Unknown Source) ~[?:1.8.0_271] {}
	at java.util.concurrent.CompletableFuture$AsyncRun.exec(Unknown Source) ~[?:1.8.0_271] {}
	at java.util.concurrent.ForkJoinTask.doExec(Unknown Source) ~[?:1.8.0_271] {}
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(Unknown Source) ~[?:1.8.0_271] {}
	at java.util.concurrent.ForkJoinPool.runWorker(Unknown Source) ~[?:1.8.0_271] {}
	at java.util.concurrent.ForkJoinWorkerThread.run(Unknown Source) ~[?:1.8.0_271] {}
```

</details>

**Comment by IchigoGames** [comment]
can also confirm this https://pastebin.com/VUjMkTZT after updating IE

**Comment by IchigoGames** [comment]
> You have to downgrade your IE version, not currently working with the newest update.

why not tho if its a stable build why not update IP

**Comment by adriantomaszewski** [comment]
https://pastebin.com/m2fQs0kG Can confirm 100%.

**Comment by ApexModder** [comment]
Also getting this same crash https://paste.md-5.net/apapujajux.rb

**Comment by TwistedGate** [comment]
### Due to changes done to `IETemplateMultiblock` in IE, any recent versions of IE with those changes included causes this crash.

If your crash report contains
**`java.lang.NoSuchMethodError: blusunrize.immersiveengineering.common.blocks.multiblocks.IETemplateMultiblock`**
anywhere

then the only thing i can say for the time being is to either *downgrade IE* or *remove IP*

**Comment by TwistedGate** [comment]
Fixed in `1.6.4-3.1.0-2`
